prospero-cli/src/main/resources/UsageMessages.properties

```diff
@@ -1,14 +1,42 @@
-# Main Command Header
-prospero.usage.headerHeading = @|bold Welcome to Prospero CLI!|@%n
-prospero.usage.header        = %nThis tool allows you to provision instances of Wildfly and JBoss EAP application containers.%n
+#
+# This file contains strings that appear in generated usage messages.
+#
 
-usage.synopsisHeading = %nUsage: \
+#
+# Main Command Welcome Message
+#
+# This is only printed when running prospero without any parameters.
 
-# Sub-Commands
+prospero.welcomeMessage = @|bold Welcome to Prospero CLI!|@\n\
+  \n\
+  This tool allows you to provision instances of Wildfly and JBoss EAP application containers.\n
```

Comment:
Should we refer only to Wildfly in community project?

Reply:
Right.